openpype/hosts/fusion/api/__init__.py

```diff
@@ -22,17 +22,15 @@
 __all__ = [
     # pipeline
     "ls",
-
     "imprint_container",
     "parse_container",
-
     # lib
     "maintained_selection",
     "update_frame_range",
     "set_asset_framerange",
     "get_current_comp",
     "comp_lock_and_undo_chunk",
-
+    "get_comp_render_range",
```

Comment:
Let's keep the space in between for consistency's sake.

Reply:
Interesting... Black removes the new lines, didn't think about that. Will fix!